### PR TITLE
Doxygen for src/w3fb09.f src/w3fb10.f src/w3fb11.f src/w3fb12.f src/w3fc02.f (Part of #77)

### PR DESCRIPTION
**Pull request by BrianCurtis-NOAA**
Doxygen for:
- src/w3fb09.f
- src/w3fb10.f
- src/w3fb11.f
- src/w3fb12.f
- src/w3fc02.f

Part of #77

### CODE REVIEW

**Review by edwardhartnett — APPROVED** [review]

File: src/w3fb09.f [hunk 1]
```diff
@@ -1,42 +1,34 @@
...
+C> Program history log:
+C> - John Stackpole 1988-03-01
+C> - Ralph Jones 1990-04-12 Convert to cray cft77 fortran.
 C>
-C>   REMARKS: FORMULAE AND NOTATION LOOSELY BASED ON HOKE, HAYES,
-C>     AND RENNINGER'S "MAP PROJECTIONS AND GRID SYSTEMS...", MARCH 1981
-C>     AFGWC/TN-79/003
+C> @param[in] XI I coordinate of the point.
+C> @param[in] XJ J coordinate of the point; both real*4.
+C> @param[in] ALAT1 Latitude  of lower left corner of grid (point (1,1)).
```
Comment (Hang-Lei-NOAA) [inline src/w3fb09.f]: reduce one space

File: src/w3fb10.f [hunk 2]
```diff
@@ -1,80 +1,72 @@
...
-C>         FUNDAMENTAL TRIGONOMETRIC IDENTITIES ARE USED FREELY, SUCH
-C>   AS THAT COS(X) = SIN(PI/2 - X), ETC.  SEE ALMOST ANY MATHEMATICAL
-C>   HANDBOOK, SUCH AS THE C.R.C. STANDARD MATH TABLES UNDER 'RELATIONS
-C>   IN ANY SPHERICAL TRIANGLE', OR THE NATIONAL BUREAU OF STANDARDS
-C>   'HANDBOOK OF MATHEMATICAL FUNCTIONS' UNDER SECTION 4.3.149,
-C>   FORMULAS FOR SOLUTION OF SPHERICAL TRIANGLES.
-C>         DOUBLE PRECISION IS USED INTERNALLY BECAUSE OF THE WIDE
-C>   RANGE OF GEOGRAPHIC VALUES THAT MAY BE USED.
+C> @note Fundamental trigonometric identities are used freely, such
+C> as that cos(x) = sin(pi/2 - x), etc.  See almost any mathematical
```
Comment (Hang-Lei-NOAA) [inline src/w3fb10.f]: reduce one space before 'See...'

File: src/w3fb11.f [hunk 3]
```diff
@@ -1,52 +1,43 @@
...
+C> - John Stackpole 1988-11-25
+C> - Ralph Jones 1990-04-12 Convert to cft77 fortran.
+C> - Ralph Jones 1994-04-28 Add save statement.
 C>
-C>   REMARKS: FORMULAE AND NOTATION LOOSELY BASED ON HOKE, HAYES,
-C>     AND RENNINGER'S "MAP PROJECTIONS AND GRID SYSTEMS...", MARCH 1981
-C>     AFGWC/TN-79/003
+C> @param[in] ALAT Latitude in degrees (negative in southern hemis).
+C> @param[in] ELON East longitude in degrees, real*4.
+C> @param[in] ALAT1 Latitude  of lower left point of grid (point (1,1)).
```
Comment (Hang-Lei-NOAA) [inline src/w3fb11.f]: reduce one space before 'of...'

File: src/w3fb11.f [hunk 4]
```diff
@@ -1,52 +1,43 @@
...
+C> @param[in] ELON1 Longitude of lower left point of grid (point (1,1))
+C> all real*4.
+C> @param[in] DX Mesh length of grid in meters at tangent latitude.
+C> @param[in] ELONV The orientation of the grid.  i.e.,
```
Comment (Hang-Lei-NOAA) [inline src/w3fb11.f]: reduce one space before 'i.e.'

File: src/w3fb11.f [hunk 5]
```diff
@@ -1,52 +1,43 @@
...
+C> the east longitude value of the vertical meridian
+C> which is parallel to the y-axis (or columns of
+C> of the grid) along which latitude increases as
+C> the y-coordinate increases.  real*4
```
Comment (Hang-Lei-NOAA) [inline src/w3fb11.f]: reduce one space before 'real*4'

File: src/w3fb12.f [hunk 6]
```diff
@@ -1,60 +1,53 @@
...
-C>                         IN THE OPEN SPACE WHERE THE CONE IS CUT.
-C>                  IF IERR.GE.1 THEN ALAT=999. AND ELON=999.
+C> @param[in] XI I coordinate of the point  real*4
+C> @param[in] XJ J coordinate of the point  real*4
+C> @param[in] ALAT1 Latitude of lower left point of grid (point 1,1)
+C> latitude <0 for southern hemisphere; real*4
+C> @param[in] ELON1 Longitude of lower left point of grid (point 1,1)
+C> east longitude used throughout; real*4
+C> @param[in] DX Mesh length of grid in meters at tangent latitude
+C> @param[in] ELONV The orientation of the grid.  i.e.,
```
Comment (Hang-Lei-NOAA) [inline src/w3fb12.f]: reduce one space

File: src/w3fb12.f [hunk 7]
```diff
@@ -1,60 +1,53 @@
...
+C> the east longitude value of the vertical meridian
+C> which is parallel to the y-axis (or columns of
+C> the grid) along which latitude increases as
+C> the y-coordinate increases.  real*4
```
Comment (Hang-Lei-NOAA) [inline src/w3fb12.f]: reduce one space

File: src/w3fb12.f [hunk 8]
```diff
@@ -1,60 +1,53 @@
...
+C> set negative to indicate a
+C> southern hemisphere projection; real*4
 C>
-C>   REMARKS: FORMULAE AND NOTATION LOOSELY BASED ON HOKE, HAYES,
-C>     AND RENNINGER'S "MAP PROJECTIONS AND GRID SYSTEMS...", MARCH 1981
-C>     AFGWC/TN-79/003
+C> @param[out] ALAT Latitude in degrees (negative in southern hemi.)
+C> @param[out] ELON East longitude in degrees, real*4
+C> @param[out] IERR
+C> - .eq. 0   if no problem
```
Comment (Hang-Lei-NOAA) [inline src/w3fb12.f]: reduce one space

File: src/w3fb12.f [hunk 9]
```diff
@@ -1,60 +1,53 @@
...
+C> - .ge. 1   if the requested xi,xj point is in the
```
Comment (Hang-Lei-NOAA) [inline src/w3fb12.f]: reduce one space

**Review by Hang-Lei-NOAA — APPROVED** [review]

**Pull request merged**